forUncrustifySources.cfg

```diff
@@ -75,21 +81,31 @@ nl_enum_identifier_colon                  = remove
 nl_enum_leave_one_liners                  = true
 nl_fcall_brace                            = add
 nl_fdef_brace                             = force
+nl_fdef_brace_cond                        = force
 nl_finally_brace                          = add
 nl_for_brace                              = add
+nl_func_call_empty                        = remove
 nl_func_call_paren                        = remove
+nl_func_call_paren_empty                  = remove
```

Comment:
BTW, this is superfluous with `nl_func_call_paren`. (Apologies, I missed this one in my previous list; mea culpa.)